### DISCUSSION TITLE
Talents & Items

### DISCUSSION THREAD

**Discussion by GiantDwarf0** [FakeFishGames/Barotrauma#10295] [Unstable feedback]
So far I love the items, adding variety and alternatives to current metas while also changing the typical persons progression. Honestly my only gripe so far being that the exosuit doesn't really feel like an exosuit should, it looks more like the model was enlarged rather than something you're wearing, or more accurately piloting. And while I love the items, I cannot say that the talent changes are in good taste; and personally I think its is both a bad direction for the game to head in, and also will likely determine its future success. The lack of choice or nuance feels uncharacteristic, I mean look how complex wiring can get but talents had to be simplified? I understand needing to help newcomers but I had just introduced my friend to the game and he took off running with it and is doing much better in his first hours than me on my 50th, with only my guidance. 

As for balance I feel there are two paths, either to chose to play it the more player ergonomic route; that is to buff late-to-mid game hazards so that they can threaten meta builds, like make it so there is a combination of heavy bite wounds with radiation poisoning, dealing with pucs by physical wounds, or body armors with radiation, and similar concepts. You can also do the other viable option of simply adding weaknesses where there were none, such as making all genes have prebuilt weakness aside from genetic taint, like burns causing double or triple damage to matriarch genes and perhaps stopping them from healing without medical aid. You may also add enemies in EOD style armor or chitin plates that is perfectly built to deal with blasts of explosive slugs. Swarms of small enemies like the bitters in ruins also help add panic and ammo usage where focus fire of eight people can delete almost any encounter, and ceaseless hordes would be great for horror while not being susceptible to mass single target fire as is whats mostly needed late game.

But ignoring what we could do, I say sticking with the current system is better, after all if we want player choice why are we eliminating a majority of the choices players are making currently? to go with the vision wouldn't it make more sense to add more levels? more perks? Each level and choice can mean less but they add up to a similar result, more choices, more agency or at least more interesting options to take (and with strategic placement in each tree you can still prevent OP combinations happening without multiple high level people coordinating). I feel like this update could be very divisive when it comes out, and I can only hope you can make a combination of your vision for the game and the communities hopes so we both end with our own little thalassophobic horror paradise.

**Comment by Ek-Videogames** [comment]
Yeah talents were a novel idea but the direction they are going in now progression-wise has me unimpressed.
